<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REIXSSampleChamberButtonPanel</class>
 <widget class="QWidget" name="REIXSSampleChamberButtonPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="sampleLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Top View (X, Y, Î¸)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="sampleLabel2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="3">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="3">
    <widget class="QLabel" name="loadLockLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load Lock</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="AMControlMoveButton" name="sampleCW">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CW</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/arrow-CW.png</normaloff>:/22x22/arrow-CW.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AMControlMoveButton" name="sampleXup">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+X</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-up-dark.png</normaloff>:/22x22/go-up-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="AMControlMoveButton" name="sampleCCW">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CCW</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/arrow-CCW.png</normaloff>:/22x22/arrow-CCW.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="AMControlMoveButton" name="sampleZup">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+Z</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-up-dark.png</normaloff>:/22x22/go-up-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="AMControlMoveButton" name="loadLockCW">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CW</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/arrow-CW.png</normaloff>:/22x22/arrow-CW.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="AMControlMoveButton" name="loadLockZup">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+Z</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-up-dark.png</normaloff>:/22x22/go-up-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="AMControlMoveButton" name="loadLockCCW">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CCW</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/arrow-CCW.png</normaloff>:/22x22/arrow-CCW.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="AMControlMoveButton" name="sampleYdown">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-Y</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-previous-dark.png</normaloff>:/22x22/go-previous-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="AMControlMoveButton" name="sampleXdown">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-X</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-down-dark.png</normaloff>:/22x22/go-down-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="AMControlMoveButton" name="sampleYup">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+Y</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-next-dark.png</normaloff>:/22x22/go-next-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="AMControlMoveButton" name="sampleZdown">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-Z</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-down-dark.png</normaloff>:/22x22/go-down-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="AMControlMoveButton" name="loadLockZdown">
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>46</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-Z</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/22x22/go-down-dark.png</normaloff>:/22x22/go-down-dark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="9">
    <widget class="QToolButton" name="stopAll">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AMControlMoveButton</class>
   <extends>QToolButton</extends>
   <header>ui/beamline/AMControlMoveButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
